extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -295,9 +295,9 @@ export class AmpAnalytics extends AMP.BaseElement {
     );
 
     this.transport_.maybeInitIframeTransport(
-      this.getAmpDoc().win,
+      this.win,
```

Comment:
@choumx this change might have caused a regression. any reason to not use `getAmpdoc`?

Confusingly enough, in the case of FIE rendering, `getAmpDoc() != this.win`, 
(although @dvoytenko is fixing that).

There we want to point to the top window, not the FIE window.

Reply:
Fix PR is here: #25917

Reply:
Thanks for fixing this.